code/game/atoms.dm

```diff
@@ -649,4 +649,20 @@ its easier to just keep the beam vertical.
 	testing("Set [src]'s gender to [gend], old gender [old_gender] previous gender [prev_gender]")
 
 /atom/proc/mop_act(obj/item/weapon/mop/M, mob/user)
-	return 0
\ No newline at end of file
+	return 0
+
+/atom/proc/get_areaa()
+
+/area/get_areaa()
+	return src

```

Comment:
Don't we intentionally avoid doing it this way due to lighting subareas messing up areas? Or is my understanding of area code missing something